x/auth/client/cli/tx_sign.go

```diff
@@ -303,16 +295,17 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		defer fp.Close()
+		if err = fp.Close(); err != nil {
+			return err
+		}
 
-		return clientCtx.PrintString(fmt.Sprintf("%s\n", json))
+		return clientCtx.PrintBytes(json)
 	}
 }
 
-func marshalSignatureJSON(txConfig client.TxConfig, txBldr client.TxBuilder, generateSignatureOnly bool) ([]byte, error) {
+func marshalSignatureJSON(txConfig client.TxConfig, txBldr client.TxBuilder, signatureOnly bool) ([]byte, error) {
```

Comment:
This just increases the size of the diff. Please revert this change and the following one.

Reply:
This argument name doesn't make sense it this function - we are not generating signature, we only want to marshal the signature here.

Reply:
Fair enough. Leave it